src/Certify.Tests/Certify.Core.Tests.Unit/CertifyManagerTests.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Threading.Tasks;
+using Certify.Config.Migration;
+using Certify.Management;
+using Certify.Models;
+using Certify.Service;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Certify.Core.Tests.Unit
```

Comment:
This is probably an integration test because the defaults will be real data store, real accounts etc. If we were injecting or replacing the respective providers with test ones it might be a unit test. I realize though that there are already plenty of tests in Unit that also fail to make that distinction, but perhaps going forward we should be more pedantic about the distinction between Unit and Integration tests.

Reply:
This has been moved to Integration tests